New google cloud broker (#106)
kyma-project/addons: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/plans-details.md

---
title: Details
type: Details
---

## Service description

The `GCP Service Broker` service provides the following plan names and descriptions:

| Plan Name | Description |
|-----------|-------------|
| `Default` | Installs the Google Cloud Platform (GCP) Service Broker in a default configuration. |

> ### Provisioning parameters

Reviewer: ```suggestion
## Provisioning parameters
```

> These are the provisioning parameters for the given plans:

Reviewer: ```suggestion
These are the provisioning parameters for the `Default` plan:
```

> #### Default plan

Reviewer: ```suggestion

```
